docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
         - SENTRY_ENVIRONMENT
         - SENTRY_ORG
         - SENTRY_PROJECT
+        - PAYLOAD_PUBLIC_APP_URL
```

Comment:
`docker-compose` is local dev only & I don't think we use 2 separate domain when doing local development